MAE-173: Initialized extension structure

nishant-bhorodia opened the pull request.

nishant-bhorodia committed:
MAE-173: Initialized extension structure

omarabuhussein commented on info.xml:
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<extension key="uk.co.compucorp.membershipextrasdefaultconfig" type="module">
+  <file>membershipextrasdefaultconfig</file>
+  <name>Membership Extras Default Config</name>
```
can you change it to **: Membership Extras Default Configurations**

nishant-bhorodia committed:
MAE-173: Changed name of extension.

Changed name of extension from Membership Extras Default Config to Membership Extras Default Configurations in info.xml
